kart/kartapi.py

```diff
@@ -57,7 +57,7 @@ def inner(*args):
             lines = str(ex).splitlines()
             msglines = []
             for line in lines:
-                if line.startswith("ERROR 1: Can't load"):
+                if line.startswith("ERROR 1: Can't load") or '.dylib' in line:
```

Comment:
can we add a comment for future maintainers with examples of the messages we're trying to hide?

Reply:
Yep. I don’t like this as gov approach so much, needs a bit of a rethink as well and a better dialog for the user.

Reply:
Presumably there's some environment something QGIS is passing to Kart which leads to messages? Ideally we fix that.